drivers/max31855/max31855_saul.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2024 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_max31855
+ * @{
+ *
+ * @file
+ * @brief       SAUL adaption for MAX31855 devices
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <string.h>
+
+#include "saul.h"
+#include "phydat.h"
+#include "max31855.h"
+
+static int read_temp(const void *dev, phydat_t *res, bool thermocouple)
+{
+    max31855_data_t data;
+    max31855_read((max31855_t *)dev, &data);
+
+    if (data.fault != MAX31855_FAULT_NO_FAULT) {
+        return -ECANCELED;
+    }
+
+    res->unit = UNIT_TEMP_C;
+    res->scale = -2;
```

Comment:
Whats this?

Reply:
This is phydat's way of saying that the value is in the scale 10^-2